upt_macports/templates/ruby.Portfile

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           ruby 1.0
+{% endblock %}
+{% block nameversion %}
+ruby.setup          {{ pkg._pkgname() }} {{ pkg.upt_pkg.version }} gem {} rubygems
```

Comment:
based on a quick glance through the PortGroup I don't really understand the need for the ```{}```.... can you please check?